recipEasy/.env

```diff
@@ -0,0 +1,2 @@
+APIKEY='584a968330c9a233c596f56784cf5806'
```

Comment:
You should not commit your env file. Because this contains private information that the public should not see. 
Add this to your gitignore file. 

The way to introduce these variables in your pipeline is by using GitHub Secrets.

Reply:
Ok, will move it to gitignore. 
 Last night when I imported the .env file using Vites "meta.import...", when I made the call to the API, it said my apikey was undefined and so was the API id. kind of confused as to why. 😅

Reply:
Did you import the env variables ?